src/main/scala/com/horizon/exchangeapi/tables/Agbots.scala

```diff
@@ -190,7 +190,6 @@ class AgbotMsgs(tag: Tag) extends Table[AgbotMsgRow](tag, "agbotmsgs") {
   def timeExpires = column[String]("timeexpires")
   def * = (msgId, agbotId, nodeId, nodePubKey, message, timeSent, timeExpires) <> (AgbotMsgRow.tupled, AgbotMsgRow.unapply)
   def agbot = foreignKey("agbot_fk", agbotId, AgbotsTQ.rows)(_.id, onUpdate=ForeignKeyAction.Cascade, onDelete=ForeignKeyAction.Cascade)
-  def node = foreignKey("node_fk", nodeId, NodesTQ.rows)(_.id, onUpdate=ForeignKeyAction.Cascade, onDelete=ForeignKeyAction.Cascade)
```

Comment:
I would suggest just commenting out this line instead of deleting it. And leave a comment something like: we can't have a foreign key for the node, because if a node is deleted during unregistration, the agbot needs to have a chance to process the agreement cancellation msg.

I often leave a line of code around, commented out, when that line would be the obvious thing to do, but there is a very specific reason why we can't. So the next guy doesn't make the same mistake.